src/config/main-config.js

```diff
@@ -0,0 +1,19 @@
+require('dotenv').config();
+const path = require('path');
+const logger = require('morgan');
+const cors = require('cors')
+const bodyParser = require('body-parser');
+
+module.exports = {
+  init(app, express) {
+    app.use(bodyParser.urlencoded({ extended: true }));
+    app.use(logger('dev'));
+    app.use(express.json());
```

Comment:
Huh, I wasn't even aware that express had a JSON body parser bundled with it! Sounds like there was [a bit of back-and-forth](https://stackoverflow.com/a/47232318/324399) with it. We can probably do away with the `body-parser` dep in that case.

Reply:
See previous